src/logging/request-logger.js

```diff
@@ -0,0 +1,40 @@
+
+/**
+ * Module dependencies.
+ */
+
+import debugnyan from 'debugnyan';
+import { obfuscate } from './request-obfuscator';
+import request from 'request';
+import requestLogger from '@uphold/request-logger';
+
+/**
+ * Instances.
+ */
+
+const log = debugnyan('bitcoin-core:request');
+
+/**
+ * Default logging function.
+ */
+
+function defaultLogging(data) {
+  const request = data;
+
+  if (request.type === 'response') {
```

Comment:
Perhaps logging requests with binary data (`application/octet-stream`) could be skipped?